_sections/120-entities.md

```diff
@@ -57,6 +68,49 @@ This specification is a sub-specification of the [ODK XForms Specification](./).
     ...
 ```
 
+### Example of an entity-updating form
+
+```xml
+<?xml version="1.0"?>
+<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa" xmlns:orx="http://openrosa.org/xforms" xmlns:odk="http://www.opendatakit.org/xforms" xmlns:entities="http://www.opendatakit.org/xforms/entities">
+    <h:head>
+        <h:title>Trees circumference update</h:title>
+        <model odk:xforms-version="1.0.0" entities:entities-version="2023.1.0">
+            <instance>
+                <data id="trees_update" version="20240108145123">
+                    <tree/>
+                    <circumference/>
+                    <meta>
+                        <instanceID/>
+                        <entity dataset="trees" id="" update="1" baseVersion="">
```

Comment:
It initially surprised me that this declares `update="1"` rather than `update="true"`. Reading everything again, I get that you can use `true` or `1`, but maybe it would be better for the `create` and `update` examples to be consistent? I think otherwise there's an implication that `update` is different in someway - for a couple of seconds I was thinking that update conveyed the "next" entity version for instance.